src/EA.Weee.Web/Content/weee-page.scss

```diff
@@ -530,4 +530,8 @@ input.tonnage-entry {
     .import-icon-error-upload {
         left: 5% !important;
     }
+}
+
+.govuk-phase-tag-override {
+    background-color: #005ea5 !important;
```

Comment:
Is !important needed here? Normally bad practice to use it

Reply:
removed